test/integration/verify/controls/azure_db_migration_service.rb

```diff
@@ -0,0 +1,13 @@
+resource_group_name = attribute(:resource_group, default: nil)
+service_name = attribute(:inspec_db_migration_service_name, default: nil)
+sku_name = attribute(:inspec_db_migration_service_sku_name, default: nil)
+location = attribute(:location, default: nil)
+
+control 'azure_db_migration_service' do
+  describe azure_db_migration_service(resource_group: resource_group_name, service_name: service_name) do
```

Comment:
After the control block, we should add the impact and the description. This is how we do in was. Do we have to do the same?

Reply:
it is not mandatory ..